Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM ubuntu:focal as app
+MAINTAINER sre@edx.org
+
+
+# Packages installed:
+
+# language-pack-en locales; ubuntu locale support so that system utilities have a consistent
+# language and time zone.
+
+# python; ubuntu doesnt ship with python, so this is the python we will use to run the application
+
+# python3-pip; install pip to install application requirements.txt files
+
+# libmysqlclient-dev; to install header files needed to use native C implementation for
+# MySQL-python for performance gains.
+
+# libssl-dev; # mysqlclient wont install without this.
+
+# python3-dev; to install header files for python extensions; much wheel-building depends on this
+
+# gcc; for compiling python extensions distributed with python packages like mysql-client
+
+# If you add a package here please include a comment above describing what it is used for
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -qy install --no-install-recommends \
+ language-pack-en locales \
+ python3.8 python3-dev python3-pip \
+ # The mysqlclient Python package has install-time dependencies
+ libmysqlclient-dev libssl-dev pkg-config \
+ gcc
+
+
+RUN pip install --upgrade pip setuptools
+# delete apt package lists because we do not need them inflating our image
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN ln -s /usr/bin/python3 /usr/bin/python
+
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+ENV DJANGO_SETTINGS_MODULE sanctions.settings.production
+
+EXPOSE 18770
+RUN useradd -m --shell /bin/false app
```

Comment:
I think you'll need to add a line between 44 and 45 that says:
```
EXPOSE 18771
```
similar to https://github.com/edx/subscriptions/blob/main/Dockerfile#L81

I believe this is because 18770 will be the HTTP port, and 18771 will be the HTTPS port